src/applications/vaos/components/AppointmentRequestListItem.jsx

```diff
@@ -3,15 +3,12 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import AdditionalInfo from '@department-of-veterans-affairs/formation-react/AdditionalInfo';
 
-import {
-  getLocationHeader,
-  getAppointmentLocation,
-  getRequestDateOptions,
-  getRequestTimeToCall,
-  getPurposeOfVisit,
-  getAppointmentTypeHeader,
-  sentenceCase,
-} from '../utils/appointment';
+import { formatBestTimetoCallForList, sentenceCase } from '../utils/formatters';
```

Comment:
These two formatters aren't used anywhere else.  What do you think about moving them into this component?

Reply:
Hmm. That's a good point. I think generic formatters like `sentenceCase` should live in `formatters`, but something like `formatBestTimetoCallForList`, if it's only used once, should probably live in the component wher it's used.